posthog/api/early_access_feature.py

```diff
@@ -203,6 +203,7 @@ def create(self, validated_data):
                     "key": feature_flag_key,
                     "name": f"Feature Flag for Feature {validated_data['name']}",
                     "filters": filters,
+                    "created_from": "early_access_features",
```

Comment:
I recently used `creation_context` elsewhere for this kind of thing... I'm not sure if that's a better property name, but might be worth being consistent. Up to you though!

Reply:
lol I literally debated this for a while before choosing "created_from" (I also consider "creation_source").  I'll use `creation_context` to be semi-consistent.